source/beamline/IDEAS/g13ElementDetailedDetectorView.h

```diff
@@ -0,0 +1,34 @@
+#ifndef G13ELEMENTDETAILEDDETECTORVIEW_H
+#define G13ELEMENTDETAILEDDETECTORVIEW_H
+
+#include "ui/IDEAS/IDEASXRFDetailedDetectorViewWithSave.h"
+#include "beamline/IDEAS/IDEAS13ElementGeDetector.h"
+
+class g13ElementDetailedDetectorView : public IDEASXRFDetailedDetectorViewWithSave
+{
+	Q_OBJECT
+public:
+	/// Constructor
+	g13ElementDetailedDetectorView(IDEAS13ElementGeDetector *detector, QWidget *parent = 0);
+
+	/// Destructor
+	virtual ~g13ElementDetailedDetectorView();
+
+	/// Re-implementing to add peaking time spin box
+	virtual void buildDetectorView();
+
+public slots:
+	/// Handles setting the peaking time.
+	void onPeakingTimeChanged();
```

Comment:
There is a naming convention that we can talk about some time for slots like these.  They are handy for identifying what the slot is doing and what caused it to trigger.